gitops/addons/charts/cw-prometheus/templates/prometheus-eks.yaml

```diff
@@ -547,7 +234,3 @@ spec:
             name: prometheus-config
       terminationGracePeriodSeconds: 60
       serviceAccountName: cwagent-prometheus
-      {{- with .Values.tolerations }}
-      tolerations:
```

Comment:
We added this to the helm chart to allow to put this agent on managed node groups and leave karpenter nodes only for workoads.